std/fmt/index.zig

```diff
@@ -892,9 +897,150 @@ test "fmt.format" {
     }
     {
         const value = @intToPtr(*i32, 0xdeadbeef);
+        // Print Pointer Path3
         try testFmt("pointer: i32@deadbeef\n", "pointer: {}\n", value);
         try testFmt("pointer: i32@deadbeef\n", "pointer: {*}\n", value);
     }
+    {
+        const value: [3]u8 = "abc";
+        try testFmt("const [3]u8: abc\n", "const [3]u8: {}\n", value);
+    }
+    {
+        var value = "abc";
+        try testFmt("var []u8: abc\n", "var []u8: {}\n", value);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        const value = "abc";
+
+        const expected = try bufPrint(buf2[0..], "&value: [3]u8@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf1[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: u1 = 1;
+
+        const expected = try bufPrint(buf1[0..], "&value: u1@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: u8 = 8;
+
+        const expected = try bufPrint(buf1[0..], "&value: u8@{x}", @ptrToInt(&value));
+        const actual = try bufPrint(buf2[0..], "&value: {*}", &value);
+        try testExpectedActual(expected, actual);
+    }
+    {
+        var buf1: [32]u8 = undefined;
+        var buf2: [32]u8 = undefined;
+        var value: u127 = 127;
+
+        const expected = try bufPrint(buf1[0..], "&value: u127@{x}", @ptrToInt(&value));
```

Comment:
what's with all these weird integer size tests? what do they bring to the table? we already have tests for integers.

Reply:
I thought it a good idea to test the extremes of printing bit fields, I can remove it you prefer.